### DISCUSSION TITLE
Elementor style code is in RSS feed

### DISCUSSION THREAD

**Discussion by TimBuckToo** [elementor/elementor#24325] [Developers Community]
I'm having trouble with my posts showing up in the /feed/ with Elementor code in the content. See the attached screenshot for an example. How do I remove that bit of styling code from the RSS? I looked all over and chatted with an Elementor rep, and still no answers.

**Comment by theavic** [comment]
> I'm having trouble with my posts showing up in the /feed/ with Elementor code in the content. See the attached screenshot for an example. How do I remove that bit of styling code from the RSS? I looked all over and chatted with an Elementor rep, and still no answers.

I think you may have missed the attached screenshots

**Reply by TimBuckToo** [reply]
Thank you for letting me know. Here they are.
![2023-11-17 14_10_10-christianaidministries org_feed_ - Brave](https://github.com/elementor/elementor/assets/135131070/8be45848-2b9a-4a4b-9f3b-be6c7864fa79)
![2023-11-17 14_14_18-CAM Daily Emails _ Drag   drop Editor - Brave](https://github.com/elementor/elementor/assets/135131070/49b879f5-17e0-4c23-9cfa-2e0367e4cc3c)

**Comment by TimBuckToo** [comment]
Also, to add more context to this, I disabled all plugins besides Elementor & Elementor Pro and it still put the extra Elementor styling code into my feed. The minute I disable Elementor, it of course goes away. I'm also using Hello Elementor as my theme. If I revert my post to WordPress, it also resolves the issue. I was hoping I could use Elementor templates for publishing posts, but maybe I have to rethink this despite my hopes. Any thoughts?
